src/utils/columnUtils.js

```diff
@@ -25,7 +25,14 @@ export function getColumnProperties(rowProperties, allColumns=[]) {
     // build one object that contains all of the column properties keyed by id
     children.reduce((previous, current, i) => {
```

Comment:
Would a simpler solution be to `_.flatten(children).reduce(...)` here?

Reply:
it's better!